src/views/layouts/default.pug

```diff
@@ -6,6 +6,7 @@
 block variables
 
 doctype html
+html (lang='en')
```

Comment:
There is already and html tag below, not sure if pug combines them? Probably just need to add the `(lang='en')` to the existing tag instead of adding a new one.

Reply:
Didn't see that. Will check again

Reply:
I think it's correct

Reply:
@coltonehrman  is correct, you want `html.smooth-scroll(lang="en")` rather than adding a new html tag

Reply:
I've changed this as requested.